keps/sig-cli/2590-kubectl-subresource/README.md

```diff
@@ -332,10 +340,17 @@ No.
 
 ### Monitoring Requirements
 
+Considering that this feature is a flag contained fully within in the `kubectl` client, there exist no 
+monitoring requirements for the same.
```

Comment:
This is not true, this KEP required server side changes.  For example, https://github.com/kubernetes/kubernetes/pull/103516 was added in v1.24.  From that PR, I do not see any generic integration test that was added to make sure that all `scale` sub-resources continue to support tables going forward.  This needs to be added to prevent the feature from regressing in the future for new built-in types.  This is not an issue for CRDs because they all use the same code path.  This is (generally) not an issue for the `status` sub-resource because that uses the same code path as the regular resource, but it may be good to add an integration test just in case.